bulk.go

```diff
@@ -0,0 +1,32 @@
+package couchdb
+
+import "encoding/json"
+
+type bulkID struct {
+	ID string `json:"id"`
+}
+
+type BulkGet struct {
+	Docs []struct{ ID string } `json:"docs"`
+}
+
+type errorWrapper struct {
+	ID     string `json:"id"`
+	Rev    string `json:"rev"`
+	Error  string `json:"error"`
+	Reason string `json:"reason"`
+}
+
+type docWrapper struct {
+	Ok    *json.RawMessage `json:"ok"`
```

Comment:
There's no need for this to be a reference.

Reply:
fixed 6abf3504aa3126f34fa188cfb56de4be7a193c39